runtime/binding-http/src/main/java/io/aklivity/zilla/runtime/binding/http/internal/config/HttpRouteConfig.java

```diff
@@ -77,9 +78,30 @@ public Map<String8FW, String16FW> overrides()
     }
 
     boolean authorized(
-        long authorization)
+        long authorization,
+        Function<String, String> headerByName)
     {
-        return authorized.test(authorization);
+        UnaryOperator<String> resolve = input ->
+        {
+            String result = input;
+            boolean replaced = false;
+
+            if (result.contains("${method}"))
+            {
+                result = result.replace("${method}", headerByName.apply(":method"));
+                replaced = true;
+            }
+
+            if (result.contains("${path}"))
+            {
+                result = result.replace("${path}", headerByName.apply(":path"));
+                replaced = true;
+            }
+
+            return replaced ? result : headerByName.apply(input);
```

Comment:
What is the rationale to have `headerByName.apply(input)` as the default, wouldn't it just be `input`?

Suggest we simplify the code, into prepare and execute phases. Later we might potentially be able to hoist the prepare phase so it is only done once.

```
UnaryOperator<String> resolve = input ->
{
    String format = input.replace("${method}", "%1$s").replace("${path}", "%2$s");
    return format != input
        ? format.formatted(headerByName.apply(":method"), headerByName.apply(":path"))
        : input;
}
```

Note: even if they put `"${path}-${method}"` it will still work giving `/path-GET`, even though parameters are always `:method` first then `:path` in the call to `.formatted(...)` (due to argument index in format specifier).